helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/TestWagedRebalancer.java

```diff
@@ -110,9 +111,9 @@ protected ResourceControllerDataProvider setupClusterDataCache() throws IOExcept
   }
 
   @Test
-  public void testRebalance() throws IOException, HelixRebalanceException {
+  public void testRebalance() throws IOException, HelixRebalanceException, JMException {
     _metadataStore.clearMetadataStore();
-    WagedRebalancer rebalancer = new WagedRebalancer(_metadataStore, _algorithm);
+    WagedRebalancer rebalancer = new WagedRebalancer(_metadataStore, _algorithm, null);
```

Comment:
Not sure about all the usage, but if all test code is passing null, we can just remove this parameter from this constructor.